scrapers/FreakMobMedia.yml

```diff
@@ -0,0 +1,33 @@
+name: "freakmobmedia"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - freakmobmedia.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="bodywhite"]/div[@class="title"]//h3/text()
+      Details: 
+          selector: //meta[@name="description"]/@content
+          postProcess:
+            - replace:
+                - regex: "Join FreakMob Media Now For Full Access!"
+                  with: ""
+                - regex: "JOIN NOW or BUY JUST THIS CLIP"
+                  with: ""
```

Comment:
i would suggest making the match case insensitive
and using a single regex for the cleaning up
eg
```yaml
        postProcess:
          - replace:
              - regex: "(?i)(Join FreakMob Media Now For Full Access!)|(JOIN NOW or BUY JUST THIS CLIP)"
                with: ""
```